### PR TITLE
added http request logger morgan

### PR DESCRIPTION
**Pull request by gouravcoder**

### CODE REVIEW

File: src/server.aot.ts [hunk 1]
```diff
@@ -47,6 +48,8 @@ app.set('json spaces', 2);
 app.use(cookieParser('Angular 2 Universal'));
 app.use(bodyParser.json());
 
+app.use(morgan('dev'));
```
Comment (gdi2290) [inline src/server.aot.ts]: aot is also prod can you change this to
```
app.use(morgan('common', {
  skip: (req, res) => res.statusCode < 400,
  stream: __dirname + '/morgan.log'
}));
```

Comment (gouravcoder) [inline src/server.aot.ts]: i have made necessary changes, updated pr.

**Comment by gdi2290** [comment]
@gouravcoder can you update the branch with master changes?

**Comment by gouravcoder** [comment]
@gdi2290 updated pr

**Pull request merged**